lib/suspenders/app_builder.rb

```diff
@@ -246,6 +246,10 @@ def configure_action_mailer_in_specs
       copy_file 'action_mailer.rb', 'spec/support/action_mailer.rb'
     end
 
+    def configure_capybara_webkit
+      copy_file 'capybara_webkit.rb', 'spec/support/capybara_webkit.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.